code/modules/client/preference/loadout/loadout_uniform.dm

```diff
@@ -61,17 +61,17 @@
 /datum/gear/uniform/skirt/job/viro
 	display_name = "skirt, virologist"
 	path = /obj/item/clothing/under/rank/virologist/skirt
-	allowed_roles = list("Chief Medical Officer","Medical Doctor")
+	allowed_roles = list("Chief Medical Officer","Medical Doctor" , "Virologist")
```

Comment:
Try to think in terms of what they have access to at round start in their locker; at best CMO would have access to their own skirt, but not a virologists. So this one should really only be "Virologist"